_utilities/layout-grid.md

```diff
@@ -504,116 +357,296 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
-
-#### Spacing settings
-
-{% highlight scss %}
-// Values are set as units tokens.
-
-$theme-column-gap-sm:               2px;
-$theme-column-gap-md:               2;
-$theme-column-gap-lg:               3;
-$theme-column-gap-mobile:           2;
-$theme-column-gap-desktop:          4;
-$theme-grid-container-max-width:    'desktop';
-$theme-site-margins-breakpoint:     'desktop';
-$theme-site-margins-width:          4;
-$theme-site-margins-mobile-width:   2;
-{% endhighlight %}
-
-#### Utilities settings
-
-{% highlight scss %}
-// Turn on or off breakpoints
-$theme-utility-breakpoints: (
-  'card':              false,   // 160px
-  'card-lg':           false,   // 240px
-  'mobile':            false,   // 320px
-  'mobile-lg':         true,    // 480px
-  'tablet':            true,    // 640px
-  'tablet-lg':         false,   // 880px
-  'desktop':           true,    // 1024px
-  'desktop-lg':        false,   // 1200px
-  'widescreen':        false,   // 1400px
-);
-{% endhighlight %}
-
-### Mixins
-Mixins can be used in conjunction with grid variables to add grid functionality to semantic component Sass.
-
-{% highlight scss %}
-// Creates a wrapper for a series of rows
-// $container-size can be mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, or widescreen
-@include grid-container;
-@include grid-container($container-size);
-
-// Creates a wrapper for a series of columns
-@include grid-row;
-
-// Specify the width between columns
-// $gap-size can be sm, md, lg, 2px, 05, 1, 3, 4, 6
-@include grid-gap;
-@include grid-gap($gap-size);
-
-// Make the element full-width
-@include u-width(full);
-
-// Specify the number of columns the element should span
-// $columns can be auto, fill, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12
-@include grid-col;
-@include grid-col($columns);
-
-// Get fancy by offsetting or changing the display order
-// $offset can be none, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include grid-offset($offset);
-
-// $order can be first, last, initial, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include u-order($order);
-{% endhighlight %}
-
-### Example usage
-You can modify the variables to your custom values or just use the mixins with their default values. Here’s an example of using the default settings to create a two-column layout with a gap.
+## Theme settings
+You can redefine column gap token sizes, set the grid container max width, and update site margin by defining the relevant theme setting in your `_uswds-theme.scss` file.
 
-{% highlight scss %}
-.example-container {
-  @include grid-container;
-}
+Visit the [settings page]( {{ site.baseurl }}/documentation/settings) for more information and available settings.
```

Comment:
## Theme settings section

In an effort to replace the theme settings section:

![image](https://github.com/uswds/uswds-site/assets/25211650/408d0bf3-481a-48a8-9637-c5749e162758)

I added this basic outline of the settings available to users before pointing users to the settings page for more information. 

While other utility pages do not include this section, I saw some value in maintaining the information instead of removing it completely. If we decide we don't need it, we can remove it entirely.

Reply:
Removed this section